delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/routes/StoragesRoutes.scala

```diff
@@ -100,20 +79,6 @@ final class StoragesRoutes(
                   }
                 }
               },
-              (pathPrefix("caches") & pathEndOrSingleSlash) {
```

Comment:
This was used in a debugging purpose to know what was in the distributed cache that went away